addon/templates/components/bootstrap-power-select.hbs

```diff
@@ -1,6 +1,7 @@
 {{#bootstrap/control-wrapper errors=errors classNames=classNames ariaLabelledBy=true srOnly=srOnly label=label as |control|}}
+  {{control.label elementId=control.inputId srOnly=srOnly label=label}}
```

Comment:
Do we need to pass label above and here? Shouldn't we be able to just have `{{control.label}}`? It seems like a bunch of duplication.